samples/unit-tests/color/pattern-fill/demo.js

```diff
@@ -708,4 +708,143 @@ QUnit.test('Destroy and recreate chart', function (assert) {
 
 QUnit.test('#14765: Global patterns', assert => {
     assert.ok(Highcharts.patterns, 'Global patterns should be defined');
+});
+
+function testPatternfillForChart(assert, chart) {
+    const points = chart.series[0].points,
+        customPattern = doc.getElementById(
+            getPointFillId(points[1])
+        ),
+        defaultPattern = doc.getElementById(
+            getPointFillId(points[0])
+        );
+
+    assert.ok(
+        customPattern.hasAttribute('patternTransform'),
+        'custom pattern element doesn\'t have patternTransform attribute'
+    );
+    assert.ok(
+        customPattern.getAttribute('patternTransform').includes('scale'),
+        'custom pattern; patternTransform attribute should contain scale'
+    );
+    assert.ok(
+        defaultPattern.hasAttribute('patternTransform'),
+        'custom pattern element doesn\'t have patternTransform attribute'
+    );
+    assert.ok(
+        defaultPattern.getAttribute('patternTransform').includes('scale'),
+        'default pattern; patternTransform attribute should contain scale'
+    );
```

Comment:
We have some unwritten conventions for assertion messages:
* Upper case first (real sentence)
* Always include the word "should" in some form. It ensures that we describe what whe _expect_ to see, not what we are actually looking at, which will be confusing depending on whether the test fails or passes, or if the assertion is `ok` or `notOk`.